src/libraries/System.Net.Http/src/System/Net/Http/HttpTelemetry.cs

```diff
@@ -93,6 +156,27 @@ protected override void OnEventCommand(EventCommandEventArgs command)
                 {
                     DisplayName = "Current Requests"
                 };
+
+                _totalHttp11ConnectionsCounter ??= new PollingCounter("http11-connections-current-total", this, () => Interlocked.Read(ref _openedHttp11Connections))
+                {
+                    DisplayName = "Current Http 1.1 Connections"
+                };
+
+                _totalHttp20ConnectionsCounter ??= new PollingCounter("http20-connections-current-total", this, () => Interlocked.Read(ref _openedHttp20Connections))
+                {
+                    DisplayName = "Current Http 2.0 Connections"
+                };
+
+                _totalHttp20StreamsCounter ??= new PollingCounter("http20-streams-current-total", this, () => Interlocked.Read(ref _openedHttp20Streams))
+                {
+                    DisplayName = "Current Http 2.0 Streams"
+                };
```

Comment:
An HTTP/2 stream is the same as a request.  Do we need this counter?  And if so, what's the 1.1 equivalent?

Reply:
I see this counter isn't actually listed in David's tracking issue (#37428), removed.

I added it coming from discussions around the usefulness of "single-pool-max" counters in #38686.

I realize such counters ("http11-current", "http20-current") can be implemented in a much simpler manner than was done in this PR, as we already have guarantees around Start/Stop being called on an `HttpRequestMessage` exactly once and we have the version info available there.

Reply:
> I see this counter isn't actually listed in David's tracking issue

Are there others you added that weren't previously discussed and should be removed, too?